core/new-gui/src/app/workspace/component/operator-panel/operator-label/operator-label.component.spec.ts

```diff
@@ -67,6 +72,63 @@ describe('OperatorLabelComponent', () => {
     expect(jqueryElement.data('uiDraggable')).toBeTruthy();
   });
 
+  it('should call the mouseEnter function once the cursor is hovering above a operator label', () => {
+    const spy = spyOn<any>(component, 'mouseEnter');
+    const operatorLabelElement = fixture.debugElement.query(By.css('#' + component.operatorLabelID));
+    operatorLabelElement.triggerEventHandler('mouseenter', component);
+    expect(spy).toHaveBeenCalled();
+  });
+
+  it('should call the mouseLeave function once the cursor leaves a operator label', () => {
+    const spy = spyOn<any>(component, 'mouseLeave');
+    const operatorLabelElement = fixture.debugElement.query(By.css('#' + component.operatorLabelID));
+    operatorLabelElement.triggerEventHandler('mouseleave', component);
+    expect(spy).toHaveBeenCalled();
+  });
+
+  it('should emits a command to open an tooltip instance after 500ms delay', marbles((m) => {
+    const expectedStream = m.hot('500ms -a-');
+    const actualStream = component.getopenCommandsObservable().map(() => 'a');
+    m.hot('-a-').do(() => component.mouseEnter()).subscribe();
+    m.expect(actualStream).toBeObservable(expectedStream);
+  }));
+
+  it('should display a tooltip instance with the correct content when openCommandObservable$ emits a value', marbles((m) => {
+    const operatorLabelElement = fixture.debugElement.query(By.css('#' + component.operatorLabelID));
+    m.hot('-a-').do(() => component.mouseEnter()).subscribe();
+    component.getopenCommandsObservable().subscribe(x => {
+      if (operatorLabelElement.parent) {
+        const tooltipInstance = operatorLabelElement.parent.childNodes[1].nativeNode;
+        expect(tooltipInstance.innerText).toBe(mockOperatorData.additionalMetadata.operatorDescription);
+      }
+    });
+  }));
+
+  it('should not emits a command to open tooltip instance if the cursor has left the operator label', marbles((m) => {
+    const expectedStream_1 = m.hot('-----');
+    const actualStream_1 = component.getopenCommandsObservable().map(() => 'a');
+    m.hot('-a-----').do(() => component.mouseEnter()).subscribe();
+    m.hot('---b---').do(() => component.mouseLeave()).subscribe();
+    m.expect(actualStream_1).toBeObservable(expectedStream_1);
+  }));
+
+  it('should hide the tooltip instance if cursor leaves the operator label', marbles((m) => {
+    const operatorLabelElement = fixture.debugElement.query(By.css('#' + component.operatorLabelID));
+    m.hot('-a-').do(() => component.mouseEnter()).subscribe();
+    component.getopenCommandsObservable().subscribe(x => {
+      if (operatorLabelElement.parent) {
+        const tooltipInstance = operatorLabelElement.parent.childNodes[1].nativeNode;
+        expect(tooltipInstance).not.toBeNull();
+        expect(operatorLabelElement.parent.childNodes.length).toBe(2);
+      }
+    });
```

Comment:
Same thing from above, the subscription should be moved before creating a hot observable for marbles `m.hot()...`. Also, from what I see, your test should not even enter this subscription since the `mouseEnter()` and `mouseLeave()` events are at the same time. To make sure that it doesn't open, you can have a expectation that always return false in the subscription. For instance: `expect(true).toBeFalsy()`.